### DISCUSSION TITLE
Rockets

### DISCUSSION THREAD

**Discussion by jigglypuff2731** [Slimefun-Addon-Community/Galactifun#48] [Ideas]
Rockets maybe even customs ones then need certain blocks to work and maybe a way to select the area of the rocket so when you use it to go to a planet it disappears and I know this isn’t accurate but lands back in the same place so you don’t waste a lot of stuff

**Comment by Seggan** [comment]
Juggly... since when have we released Galactifun. This is the way we currently have it.

**Reply by jigglypuff2731** [reply]
ik its not released but ideas can still be made right

**Reply by Riley31415** [reply]
rockets will be in the release yes

**Reply by jigglypuff2731** [reply]
k
